### DISCUSSION TITLE
GraphicsNode and opacity

### DISCUSSION THREAD

**Discussion by ggblatz** [css4j/echosvg#94] [Q&A] [answered]
Hi,
I use `node.paint(g)` method to display a`GraphicsNode`, but if `g` has a composite set, it seems to be ignored. However, node has a `setComposite()` method, so
 ```
node.setComposite(g.getComposite());
node.paint(g);
```
works. Are these two steps intentional?
Thanks.

**Comment by carlosame** [comment, marked as answer]
> Are these two steps intentional?

I did not write that code, but yes the composite is supposed to be explicitly set at the GraphicsNode.

Moving this to a discussion as it is not an issue.
